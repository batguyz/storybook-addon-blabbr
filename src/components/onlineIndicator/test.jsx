import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line
import Offline from './';

test('Offline indicator renderes correctly', () => {
  const tree = renderer.create(
    <Offline isOnline={false} />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Online indicator renderes correctly', () => {
  const tree = renderer.create(
    <Offline isOnline />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
