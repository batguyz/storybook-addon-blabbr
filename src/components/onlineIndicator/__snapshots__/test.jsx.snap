exports[`test Online indicator renders correctly when system is offline 1`] = `
<div
  className="blabbr-status-indicator offline"
  data-for="blabbr-indicator-tooltip"
  data-tip={true}>
  <wrapper
    className="__react_component_tooltip place-top type-dark "
    data-id="tooltip">
    
  </wrapper>
</div>
`;

exports[`test Online indicator renders correctly when system is online 1`] = `
<div
  className="blabbr-status-indicator online"
  data-for="blabbr-indicator-tooltip"
  data-tip={true}>
  <wrapper
    className="__react_component_tooltip place-top type-dark "
    data-id="tooltip">
    
  </wrapper>
</div>
`;
